pristine-zen-garden.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>CSS Zen Garden: The Beauty of CSS Design</title>
+
+		<link rel="stylesheet" media="screen" href="/214/214.css?v=8may2013">
+		<link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">
+
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta name="author" content="Dave Shea">
+		<meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+		<meta name="robots" content="all">
+	</head>
+
+<body id="css-zen-garden">
+	<div class="page-wrapper">
+		<section class="intro" id="zen-intro">
+			<header role="banner">
+				<h1>CSS Zen Garden</h1>
+				<h2>The Beauty of <abbr title="Cascading Style Sheets">CSS</abbr> Design</h2>
+			</header>
+
+			<div class="summary" id="zen-summary" role="article">
+					<p>A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets">CSS</abbr>-based design. Select any style sheet from the list to load it into this page.</p>
```

Comment:
The resource suggests that content should be indented separately from the code tags.